pkg/cloud-provider/cloudapi/aws/aws_security.go

```diff
@@ -58,18 +57,20 @@ func buildEc2UserIDGroupPairs(addressGroupIdentifiers []*securitygroup.CloudReso
 	return userIDGroupPairs
 }
 
-func buildEc2CloudSgNamesFromRules(addressGroupIdentifier *securitygroup.CloudResourceID, ingressRules []*securitygroup.IngressRule,
-	egressRules []*securitygroup.EgressRule) map[string]struct{} {
+func buildEc2CloudSgNamesFromRules(addressGroupIdentifier *securitygroup.CloudResourceID, ingressRules []*securitygroup.CloudRule,
```

Comment:
Add a comment on the function what it does

Reply:
Done